tests/utils.go

```diff
@@ -1361,6 +1363,48 @@ func RunKubectlCommand(args ...string) (string, error) {
 	return string(stdOutBytes), nil
 }
 
+func SkipIfNoOc() {
```

Comment:
Wasn't there something about using the KubeCtl path here? at least in the dev environment?

Reply:
@stu-gott This was resolved by adding the `-oc-path` flag to the

```bash
if [[ ${TARGET} == openshift* ]]; then
    oc=${kubectl}
fi

${TESTS_OUT_DIR}/tests.test -kubeconfig=${kubeconfig} -tag=${docker_tag} -prefix=${functest_docker_prefix} -oc-path=${oc} -kubectl-path=${kubectl} -test.timeout 90m ${FUNC_TEST_ARGS}
```

 `hack/functest.sh`. The value of this flag is set to empty string when the value of `TARGET` environment variable does not begin with `openshift`.